src/jobs/test-nut.yml

```diff
@@ -45,12 +49,27 @@ steps:
   - checkout
   - when:
       condition:
-        not:
-          equal: ['', <<parameters.repo_tag>>]
+        and:
+          - equal: ['windows', <<parameters.os>>]
+          - not: <<parameters.repo_tag>>
+      steps:
+        - run:
+            name: Switch to tag v<<parameters.repo_tag>> or  <<parameters.npm_module_name>>@<<parameters.repo_tag>>
+            command: |
+              git checkout v<<parameters.repo_tag>>
+              if (!$?)
+              {
+                git checkout <<parameters.npm_module_name>>@<<parameters.repo_tag>>
+              }
+  - when:
+      condition:
+        and:
+          - equal: ['linux', <<parameters.os>>]
+          - not: <<parameters.repo_tag>>
```

Comment:
same comment.  It made more sense to me when it was 
```
not:
          equal: ['', <<parameters.repo_tag>>]
```

Reply:
@mshanemc never could get that pattern to get through the pack. I'll fiddle some more.